documentation/asciidoc/computers/raspberry-pi/pcie.adoc

```diff
@@ -55,7 +55,7 @@ WARNING: The Raspberry Pi 5 is not certified for Gen 3.0 speeds. PCIe Gen 3.0 co
 
 ==== via `config.txt`
 
-The connection is certified for Gen 2.0 speeds (5 GT/sec), but you can force Gen 3.0 (10 GT/sec) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
+The connection is certified for Gen 2.0 speeds (5 GT/s), but you can force Gen 3.0 (8 GT/s) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
```

Comment:
I guess the **first** sentence here (talking about transfer speeds) is equally-applicable to both "via `config.txt`" and "via `raspi-config`" so should probably be moved to before the header?

Reply:
Actually, this is a perfect opportunity to use tabs. I'll get this fixed shortly with a new flow that uses them.

Reply:
Fixed in https://github.com/raspberrypi/documentation/pull/3872